README.md

```diff
@@ -26,7 +26,42 @@
 The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release and LTS working group's purpose is:
+
+* Management/execution the release process for all releases.
+* Maintenance of the LTS releases.
+
+Its responsibilities are:
+
+* Define the release process.
+* Generate and create releases. 
+* Manage the LTS and Current branches including backporting changes to 
+  these branches.
+
+The Release and LTS working group is structured into teams and membership in
+the working group does not automatically result in membership in these
+teams. These teams are:
+
+* Release team
+* LTS team
+* Backporting team
+
+The release team is entrusted with the secrets and CI access to be able
+build and sign releases.  **Additions to the release team must be approved
+by the CTC.**
+
+The LTS team manages the process/content of LTS releases as outlined in the
+LTS plan. All members of the working group are members of the LTS team unless
+they do not wish to be.
+
+The backporting team is the team that does the ongoing work of porting
+commits back to the LTS and Current branches. This team may consist of
+collaborators from any of the Node.js working groups. 
+
+
+## LTS Plan
```

Comment:
The LTS plan should be a separate document that is not part of the charter. That way the WG can change it when it needs without having to modify the charter.